apps/frontend/src/locales/en/translation.json

```
@@ -197,61 +197,61 @@
+    "code": "Code:",
+    "orientation": "Orientation:",
+    "editSite": "Edit site",
+    "viewFlights": "View flights",
+    "deleteSite": "Delete site"
+  },
   "settings": {
     "title": "Settings",
     "subtitle": "Configure your paragliding dashboard",
     "description": "Preferences are applied immediately to the relevant pages and kept in this browser.",
     "autoSave": "Auto-save",
     "saved": "Applied",
```

Biome (2.4.15)

[error] The key saved was already declared.

(lint/suspicious/noDuplicateObjectKeys)